### DISCUSSION TITLE
Change Key binding

### DISCUSSION THREAD

**Discussion by krauol** [opensteno/plover#1677] [General]
Hello, 

I am quite new to Plover and Steno and I wondered why there are two "S-" on the left handside of the Steno. I also want to primary learn to Steno write in german and the only dictonary I found uses "Z-" on the left handside - that makes the questions why there are two "S-" even more interesting. I already tried to change one of the "S-" under "Configure" but I am very limited in the options - "Z-" is none of them. I am using the machine "Uni v4". 
Thanks in advance!

**Comment by aerickt** [comment]
Traditional steno machines had one double height `S-` key on the left. Hobbyist steno keyboards are basically just custom mechanical keyboards, so double height keys would require extra components (stabilizers and different width key caps). Almost all hobbyist keyboard manufacturers decided it's much easier to just have two `S-` keys on the left instead.

Some people have used it for different purposes, though. I personally map it to the number key since I think it's better than having a number bar or thumber keys. Some people install the `plover-extended-stenotype` plugin which gives them extra keys they can map it to.

For German, I'm assuming you're looking at the Regenpfeifer system. You have to install the system plugin, which adds the `Z` key and will automatically remap it.

**Reply by krauol** [reply]
I got it. Thank you for you help!
